api4/team.go

```diff
@@ -996,6 +1097,7 @@ func inviteUsersToTeam(c *Context, w http.ResponseWriter, r *http.Request) {
 		}
```

Comment:
This function is tricky, because we have some partial success, with graceful we have something like "successed invites", "failed invites", and we are no tracking here the invite_id generated (which can be interesting for cross that information with the inviteId used when somebody joins a team using an invite).

Reply:
Good point @jespino .  I've added all the attempted ids, and then all the error'ed ids when failing gracefully.  https://github.com/mattermost/mattermost-server/pull/13967/commits/707fc3bcdfbc4d8bc4a3db1f710f96fd83311763

Reply:
I do not have access to the invite_id that I can see.  A separate PR will have to bubble that up from the app layer.